.nycrc.json

```diff
@@ -2,5 +2,6 @@
   "extends": "@istanbuljs/nyc-config-typescript",
   "all": true,
   "reportDir": "./test-coverage",
-  "reporter": ["lcov"]
+  "reporter": ["lcov"],
+  "exclude": "github-actions"
```

Comment:
Excluding the github-actions from test coverage. 1. Because it is CI/CD and 2. the presence of the lib an dist directories was confusing nyc.